src/components/login-form/LoginForm.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Button, Col, Form, Input, Row } from 'antd';
+import styled from 'styled-components';
+import useWindowDimensions, {
+  WindowTypes,
+} from '../../components/window-dimensions';
+
+const formHalfItemSpan = 8;
+const offsetSpan = 1;
+
+const formLayout = {
+  wrapperCol: { span: 17 },
+};
+
+interface LoginFormProps {
+  readonly onFinish: any;
+}
+
+const LoginButton = styled(Button)`
+  width: 96px;
+  margin-top: 20px;
+`;
+
+const LoginForm: React.FC<LoginFormProps> = ({ onFinish }) => {
+  const { windowType } = useWindowDimensions();
+
+  if (windowType === WindowTypes.Mobile) {
```

Comment:
In this case, I think a `switch / case` statement would be more readable. It explicitly shows which types correspond to which code.